src/styles/_calendar.scss

```
@@ -0,0 +1,22 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $calendar-width;
+
+  &__day {
+    @include cell($cell-size);
+    border: 1px solid #000;
+    background-color: $cell_color;
+    transition: background-color 0.5s ease-in-out, transform 0.5s ease-in-out;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      background-color: pink;
```

don't forget to create variables for all colors

```suggestion
      background-color: $c-pink;
```